meetings/2023-03-08.md

```diff
@@ -41,45 +48,79 @@
+* SC: Do continue to use "pod" if not in specifications. 
+* SC: Coming back to the proposal, +1s or objections? 
+
+PROPOSAL: Merge TR/2022/protocol-20221231 ?
+
+* eP: +1 to merging. (PR has been open for over 2 months now)
+* SC: +1
+* SC: I take that as no objections. 
+
+ACTION: SC to see if there are any last minute updates not touching conformance. 
+
+* eP: Are there any prior discussions about reorganizing the spec, extracting ??? into mini-specs, or taking the storage ??? because it looks like Solid ecosystem is composed by different specs, and some parts might be replaced but storage seems stable. 
+* SC: We may have touched on editors group. There's quite a bit of material that could be its own spec, but there hasn't been any proposal or initiative to actually separate until we have more material that could go into the protocol, no need to split out at the moment. Once those issues are a bit more mature we can come back to that questions. 
+* PAC: As you may remember, there's currently a project to create a working group that will take over custody of some of the spec. The charter was discussed earlier: https://github.com/solid/solid-wg-charter . Currently only mentions ??? but also mentions it might be split into smaller parts. This will be a topic of discussion of the WG, and anyone here is welcome to apply to become a member of the WG, and if you're not still welcome to join. 
+* SC: Depending on how it unfolds, it could be carried into the ??? community group. I need to create an issue to see if work needs to advance and Solid can reuse. 
+* eP: Normative references are quite a lot. If this were to go through PR, does it require that all normative references are ??? 
+* PAC: Need to check the process.
+* TBL: Having been frustrated by things at ??? stuck because can only advance when they can be pointed to normative references ??? need to take advantage of stability when the thing you're depending on is still a PR not a recommendation, if there are changes they'll be small. The idea is that there's enough flexibility so it can be implemented. 
+* eP: ??? it would mean solid-oidc would not be picked up as working item and won't make it to TR.
+* TBL: Who's actually advancing the spec in this case? separate oidc group or external? 
+* eP: just Solid CG. 
+* TBL: it should be on the recommendation track.
+* eP: ???
+* SC: Applies to WAC, Notifications Protocol etc.
+* TBL: quite often things break because authn changes. things like WAC are very stable so i worry about stability specifically in solid-oidc. 
+* PAC: One question for TBL: if this CG were to publish a report as a final report that gets kinda "frozen", would this be enough for a normative reference or does it need a more official source?
+* TBL: CG is mainly doing client-to-client specs. Profile specs, type indexes, things the server shouldn't need to know about so no dependencies on Solid protocol.
```

Comment:
```suggestion
* SC: Applies to WAC, Notifications Protocol, etc.
* TBL: Quite often things break because authn changes. Things like WAC are very stable, so I worry about stability specifically in solid-oidc. 
* PAC: One question for TBL: if this CG were to publish a report as a final report that gets kinda "frozen", would this be enough for a normative reference, or does it need a more official source?
* TBL: CG is mainly doing client-to-client specs. Profile specs, type indexes, things the server shouldn't need to know about, so no dependencies on Solid protocol.
```